### PR TITLE
Footer Card

### PR DESCRIPTION
**Pull request by Anisospina**
## Summary

Add footer.scss
New items

## Trello Card:
https://trello.com/c/WmioMOhC/14-footer

### CODE REVIEW

File: Maquetado/scss/footer.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+footer {
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: style classes instead of html tags

File: Maquetado/scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-flow: column nowrap;
+  padding:50px 15px 15px 10px;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: there should be a space after the `:` you should also use the same paddings at each side in order to have the elements centered inside

File: Maquetado/scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-flow: column nowrap;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: just leave `flex-direction: column` `nowrap` is the default value

File: application.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/*
```
Comment (SKOLZ) [inline application.css]: this file should be deleted

File: Maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+  .text-footer {
+    color: white;
+    font-family: $font-roboto;
+    font-size: $text-big;
+  }
+
+  .copy {
+    align-self: flex-end;
+    color: white;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: color variable

**Review by SKOLZ — CHANGES_REQUESTED** [review]

File: Maquetado/scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    color: $white-snow;
+    font-family: $font-roboto;
+    font-size: $text-big;
+  }
+  
+  .copy {
+    align-self: flex-end;
+    color: $white-snow;
+    font-size: $text-small;
+    font-weight: normal;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: This is the default value, you can delete it

File: Maquetado/scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column nowrap;
+  padding: 50px 15px 15px 10px;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: You  15px in the right and 10px in the left. You need to used the same value for things to be correctly centered

File: Maquetado/scss/footer.scss [hunk 8]
```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column nowrap;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: Delete the nowrap, just leave column

**Review by SKOLZ — CHANGES_REQUESTED** [review]

File: Maquetado/scss/footer.scss (hunk 6 above)
Comment (Anisospina) [reply Maquetado/scss/footer.scss]: I'm sorry, stupidity sometimes hits me... n_n

File: Maquetado/scss/footer.scss [hunk 9]
```diff
@@ -0,0 +1,23 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 50px 15px 15px 15px;
```
Comment (SKOLZ) [inline Maquetado/scss/footer.scss]: Use the shorthand: `padding: 50px 15px 15px;`

File: Maquetado/index.html [hunk 10]
```diff
@@ -89,6 +89,12 @@ <h3 class="title-tech">Django</h3>
       </div>	
     </div>
   </div>
-  
+
+  <footer>
+    <p class="text-footer">WE ARE GAME CHANGERS</p>
```
Comment (SKOLZ) [inline Maquetado/index.html]: The uppercase should be added through css using `text-transform: uppercase;`

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**